exercises/practice/grains/grains.f90

```diff
@@ -0,0 +1,14 @@
+
+module grains
+
+  implicit none
+
+contains
+
+  ! Using double precision return type instead of int to avoid overflow in int
+  double precision function square(nr)
+    integer :: nr
+    square = -1.d0
+  end function
```

Comment:
1. I suggest moving the comment out of the stub and into the instructions
2. I don't know that I've ever seen `nr` before, but `n`, `num`, `x`, or `i` are all common variable names for an integer that doesn't need a lot of context in the name

```suggestion
  double precision function square(n)
    integer :: n
    square = -1.d0
  end function
```

Reply:
change nr to n